openforcefield/tests/test_forcefield.py

```diff
@@ -806,6 +879,70 @@ def test_parameterize_ethanol_to_parmed(self):
 
         parmed_system = forcefield.create_parmed_structure(topology, positions=pdbfile.getPositions())
 
+
+
+
+    @pytest.mark.parametrize("toolkit_registry,registry_description", toolkit_registries)
+    def test_pass_invalid_kwarg_to_create_openmm_system(self, toolkit_registry, registry_description):
```

Comment:
Do we still accept `kwargs` to `create_openmm_system()`? I thought we had deprecated that since it was so terribly dangerous!

Reply:
I agree these are dangerous, but I'm struggling to think of an alternative. 

Maybe the most direct way to remove `charge_from_molecules` is to create `LibraryChargeHandler.add_molecule()`, which takes `Molecule` objects as input and turns them into `LibraryCharge`s. The big missing piece here would just be the reverse of what we discussed elsewhere in this PR -- A way to go from `OFFMol` --> tagged SMILES. We could, by default, add the new whole-molecule SMARTS to the bottom of the hierarchy. 

Other kwargs, like `allow_nonintegral_charges` and `toolkit_registry`, will require further thought. But we could start phasing them out.